run_uh.py

```diff
@@ -176,6 +177,7 @@ def main():
 	try:
 		import yaml
 	except ImportError:
+		from horizons.i18n import gettext as _
```

Comment:
These lines don't seem to need the import, but rather need a removal of the `_()`, as other languages are only initialized on line 190 of this file in `horizons.main.start()` (which calls `horizons.engine.engine.Fife.init()`, which calls `horizons.engine.settings.Settings.apply()`, which calls `horizons.i18n.change_language()`).